src/objects/jira_base.py

```diff
@@ -148,6 +147,23 @@ def create_issue(
 
         return issue
 
+    def add_attachment_to_issue(self, issue: Issue, attachment_path: str) -> None:
+        """
+        Add and upload the attachment from attachment_path to the Jira Issue object provided.
+
+        Args:
+            issue (Issue): The Jira issue the attachment should be added to.
+            attachment_path (str): The path of the file to upload as an attachment to the issue.
+
+        Returns:
+            None
+        """
+        try:
+            self.connection.add_attachment(issue=issue.key, attachment=attachment_path)
+            self.logger.info(f"Attachment {attachment_path} has been uploaded to {issue}")
+        except JIRAError as e:
+            self.logger.exception(msg=e)
```

Comment:
maybe a more meaningful log here? 
failed to add attachment to issue with error: ex